deploy/rbac-clowdapp.yml

```diff
@@ -1762,3 +1770,7 @@ parameters:
   value: 'true'
 - name: EPH_ENV
   required: true
+- name: SENTRY_ENABLED
+  value: 'true'
+- name: SENTRY_DSN
```

Comment:
I'm not sure if these parameters _must_ be defined, but I attempted to provide some reasonable defaults here. Let me know if it makes more sense to have `SENTRY_ENABLED` set to `false` by default.

Also, I'm not sure if `SENTRY_DSN` really needs a default value because of the way it's described starting on line 196. I'm merely guessing, but I assume that the value pulled from secrets/Vault would override any default specified here.

@bennyturns Feel free to let me know if I'm crazy :)